app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingSecondScreenCompose.kt

```diff
@@ -136,16 +138,17 @@ fun OnBoardingSecondScreenCompose(
 private fun CloseButton(onCloseButtonClick: () -> Unit) {
     IconButton(
         modifier = Modifier
-            .then(Modifier.size(30.dp, 30.dp))
+            .then(Modifier.size(48.dp))
```

Comment:
`then` is of no use here since you are not passing any modifier as param to the close button.